.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: go
+go:
+  - 1.12.x
 
 script:
   - env GO111MODULE=on make ci_test
```

Comment:
I think since we're requiring go 1.12 we can remove `env GO111MODULE=on`

Reply:
I tried this and the build failed because it couldn't find dependencies. Any ideas?